src/components/login/loginForm.jsx

```diff
@@ -0,0 +1,224 @@
+    const x = document.getElementById('password');
+    if (x.type === 'password') {
+      x.type = 'text';
+    } else {
+      x.type = 'password';
+    }
+  };
+
+  const googleFun = () => {
+    window.open('https://barefoot.onrender.com/api/auth/google', '_blank');
+  };
+
+  const facebookFun = () => {
+    window.open('https://barefoot.onrender.com/api/auth/facebook', '_blank');
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    axios
+      .post('https://barefoot-nomad-v2.herokuapp.com/api/auth/login', {
+        email,
+        password,
+      })
+      .then((res) => {
+        const token = res.data.accessToken;
+        const decoded = jwtDecode(token);
+        const user = { user: decoded.user, token };
+
+        dispatch(logginUser(user));
+        dispatch(loginMode());
+      })
+      .catch((err) => {
+        const { error } = err.response.data;
+        const displayError = document.getElementsByClassName('displayer');
+        displayError[0].innerText = error;
+      });
+  };
+
+  // eslint-disable-next-line no-unused-expressions
+  authenticated.user ? navigate('/controlpannel') : null;
+  // eslint-disable-next-line no-unused-expressions
+  isLogged ? navigate('/controlpannel') : null;
+
+  const {
+    classes,
+    values,
+    touched,
+    errors,
+    isSubmitting,
+    // handleChange,
+    handleBlur,
+    // handleSubmit,
+  } = props;
+
+  return (
+    <div className={classes.container}>
+      <form onSubmit={handleSubmit} data-testid="loginForm">
+        <Card className={classes.card}>
+          <Typography
+            data-testid="title"
+            style={{
+              color: colors.primaryColor,
+              paddingLeft: '20px',
+              fontWeight: 'bolder',
+            }}
+          >
+            Sign in
+          </Typography>
+          <CardContent>
+            <p className="displayer" />
+
+            <TextField
+              id="email"
+              label="Email"
+              type="email"
+              value={values.email}
+              onChange={getEmail}
+              onBlur={handleBlur}
+              helperText={touched.email ? errors.email : ''}
+              error={touched.email && Boolean(errors.email)}
+              margin="dense"
+              variant="outlined"
+              fullWidth
+            />
+
+            <TextField
+              id="password"
+              label="Password"
+              type="password"
+              value={values.password}
+              onChange={getPassword}
+              onBlur={handleBlur}
+              helperText={touched.password ? errors.password : ''}
+              error={touched.password && Boolean(errors.password)}
+              margin="dense"
+              variant="outlined"
+              fullWidth
+            />
+            <FormControlLabel
+              control={<Checkbox defaultChecked />}
+              label="Show password"
+              onClick={showPassword}
+            />
```

Comment:
Nicely done! 
One small detail though, the logic is inverted. You can either start with:
- `Checkbox defaultChecked` in **FormControlLabel** with `type="text"` in **TextField** _or_
- `Checkbox` only in **FormControlLabel** with `type="password"` in **TextField**